src/Main/Main.cpp

```diff
@@ -211,6 +211,9 @@ char                 YT_SCRIPT[MAX_STRING];
 yt_verbose           YT_VERBOSE;
 char                 YT_FIG_BASENAME[MAX_STRING];
 int                  YT_GID_Offset[NLEVEL];
+#ifdef LIBYT_JUPYTER
+bool                 LIBYT_JUPYTER_USE_CONNECTION_FILE;
```

Comment:
- I suggest replacing `LIBYT` with `YT` to be consistent with other libyt variables.
- Add this runtime parameter to `example/input/Input__Parameter`
- Record this variable in HDF5.

Reply:
Done!